### ISSUE TITLE
Nightly test case <Project Level Image Severity Policy> failed, image should not be pulled

### ISSUE THREAD

**Issue by danfengliu** [goharbor/harbor#8298]
Nightly test case <Project Level Image Severity Policy> failed. In this case, image haproxy should not be pulled due to it's low vulnerability, but the scan result is medium, so it can be pulled.

Before this case,  there is another test case to make sure scan function is ready by command as below:
     curl -k https://harbor.vke/api/systeminfo

So, is this command is strict enough for scan criteria？

 
![image](https://user-images.githubusercontent.com/44018302/61272876-ba311a00-a7da-11e9-839f-cf51c7d895a8.png)

![image](https://user-images.githubusercontent.com/44018302/61274542-996ac380-a7de-11e9-9e1f-3bc0e06c8e79.png)

**Comment by reasonerjt** [comment]
Hi, I verified with `haproxy:latest` and I think this is probably expected.
If you look at https://hub.docker.com/_/haproxy  the `latest` tag was bumped up recently to `2.0`

So I think we should probably update the test case, two thing should be done:
1) Pin image to a digest instead of using `latest`, so when you pull the image call:
 ```
docker pull haproxy@sha256:9711ff785af1146e3cc2c0276a197e5918029cdba2d22fa6f5a31dfb8060d0bb
```
2) Update the test case to add more variation:
First set the vulnerability threshold to high vulnerability and verify the image can be pulled.  Then update the threshold to medium, and verify the image cannot be pulled.

**Comment by reasonerjt** [comment]
@danfengliu I also noticed the inconsistency between Clair and Docker hub, I opened #8368 

To fix the test case failure, could you pin the image by:
```
docker pull postgres@sha256:68b49a280d2fbe9330c0031970ebb72015e1272dfa25f0ed7557514f9e5ad7b7
```

**Comment by danfengliu** [comment]
It's fixed in #8380, about adding test steps, it will be done in other issue#8397.
